gameobjects.py

```diff
@@ -23,4 +43,47 @@ def update(self):
                 self.target = None
             else:
                 self.position += pygame.Vector2.normalize(direction) * self.speed
-                self.rect.center = self.position
+                self.rect.topleft = self.position
+
+
+class Track(pygame.sprite.Sprite):
+    def __init__(self, position, length, max):
+        super().__init__()
+        self.sprite = pygame.image.load(os.path.join('resources', 'tracks.png'))
+        self.position = position
+        self.length = length
+        self.wagons = []
+        self.max = max
+
+        self.create_rect()
+        self.create_image()
+
+    def add_wagon(self, wagon):
+        if self.full() is False:
+            self.wagons.append(wagon)
+            return True
+        return False
+
+    def create_rect(self):
+        rect = pygame.rect.Rect(self.position.x, self.position.y, self.length * self.sprite.get_width(), self.sprite.get_height())
+        self.rect = rect
+
+    def create_image(self):
+        image = pygame.Surface(self.rect.size)
+
+        for i in range(0, self.length):
+            image.blit(self.sprite, [i * self.sprite.get_width(), 0])
+        self.image = image
+
+    def next_wagon_x(self):
+        x = self.rect.width - len(self.wagons) * 60 + self.position.x - 60
```

Comment:
What does the 60 represent, can we make that a const?
Also since this is a really cryptic calculation, could you do a small comment to explain what you calculate? :)

Reply:
Also we should make the spacing depended on the max amount of cars per Track, at least that would make sense to me